config/locales/en.yml

```diff
@@ -40,18 +40,18 @@ en:
       title: "Add a family"
       submit: "Add"
     show:
-      title: Family details
+      title: "Family details"
     edit:
       title: "Update family details"
     form:
       submit: "Submit"
   groups:
     new:
-      title: "Add a new group"
+      title: "Add a group"
       start_date_in_the_past_error: "from date can not be in the past!"
       start_date_after_end_date_error: "to date must be after from date!"
```

Comment:
These two errors are not specific to only new groups. A suggestion to make them a bit more general.

Looking at Rails documentation: http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models
Section 4.5.1, a suggestion to move these messages to:
```rb
activerecord:
  errors:
    models:
      group:
        start_date_in_the_past: ...
        start_date_after_end_date: ...
```

Reply:
Done in sha 83fd409